JDI.Light/JDI.Light.Tests/Tests/Common/FileInputTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using JDI.Light.Tests.UIObjects;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using static JDI.Light.Jdi;
+
+namespace JDI.Light.Tests.Tests.Common
+{
+    [TestFixture]
+    class FileInputTests : TestBase
```

Comment:
please consider making the calass public:
![image](https://user-images.githubusercontent.com/6689884/54680443-e97c9680-4b1a-11e9-81e5-f3169d3a2b2b.png)